Optimalizáció

jkacsa01 opened the pull request:
Szevasztok!

Ebben a pull requestben mindössze csak annyi a javítás, hogy nem nézi végig az ÖSSZES folyamatot aminek a fornettigyárhoz köze van, hanem csak egyet néz át amiben az adat van (vagyis a legrégebben futó folyamat).

Így nem kell bejelentkezgetni ahhoz, hogy működjön, hanem hogyha már be vagy jelentkezve akkor is le tudja kérni.

Commit:
Szivesen

marvintheskid commented on src/main/java/azahriah/nemhibas/jdktest/Main.java:
```diff
@@ -5,33 +5,28 @@
         ProcessHandle.allProcesses()
                 .filter(proc -> proc.info().command().orElse("").toLowerCase().contains("fyremc"))
+                .sorted(Comparator.comparingDouble(proc->proc.info().startInstant().get().getNano()))
+                .limit(1)
```
```Stream#findFirst```, majd a ```Stream#forEach```-et le lehetne cserélni ```Optional#ifPresent```-re

marvintheskid replied:
\+ space a lambda arrow elé/után, hogy a formázás maradjon

Commit:
Javítás

Egy kicsit várni kell miután elindult a launcher vagy miután sikeres volt a bejelentkezés.
Meg előfordulhat ritkán, hogy csak másodszorra sikerül neki lekérni.

Commit:
Kisebb javítás
